lms/djangoapps/course_api/views.py

```diff
@@ -158,11 +162,13 @@ class CourseListView(APIView):
             ]
     """
 
-    def get(self, request):
+    pagination_class = NamespacedPageNumberPagination
+    serializer_class = CourseSerializer
+    paginate_by = 20
```

Comment:
During grooming, @aleffert expressed a preference for 20 results per page, to be tweaked as we test performance.  Our default is set to 10, and I think Akiva was concerned that 10 would require too many requests for effective infinite scroll.  (He can chime in if I'm misremembering something).  

Also, it looks like I misspoke.  The setting is `settings.REST_FRAMEWORK["PAGE_SIZE"]`.